README.md

```diff
@@ -1 +1,141 @@
-# NationsConnect
\ No newline at end of file
+# NationsConnect
+<a name="readme-top"></a>
+### NationsConnect APP USING REACT
+
+# 📗 Table of Contents
+
+- [📖 About the Project]()
+  - [🛠 Built With](#build-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Nice work so far however This project should be deployed and accessible online using [Render](https://render.com/), [Netlify](https://www.netlify.com/) or [GitHub Pages](https://pages.github.com/) then add a link to the online version of your app to your README file.

- [ ] Also in your presentation video, Kindly you should also highlight some interesting piece of code or something you built that you are very proud of.